zeppelin-web/package.json

```diff
@@ -40,7 +40,7 @@
     "angular-sanitize": "1.5.7",
     "angular-touch": "1.5.7",
     "angular-ui-ace": "0.2.3",
-    "angular-websocket": "1.1.0",
+    "angular-websocket": "^1.0.13",
```

Comment:
@Armadik Hello, Could you please leave come comments that we should keep the version with some issues? I recommend upgrading versions recently so it can be upgraded with others without any notice.

Reply:
@jongyoul 
Hi, do you need to leave a comment in package.json file with a link to the task?